doc/libstemmer_python_README

```diff
@@ -0,0 +1,84 @@
+Snowball stemming library collection for Python
+===============================================
+
+How to use library
+------------------
+
+The ``snowballstemmer`` module has two functions.
+
+The ``snowballstemmer.algorithms`` function returns a list of avilable algorithm name' string.
```

Comment:
`avilable` -> `available`.

Also I think some text has been lost or damaged around `name' string` - I guess it's trying to say something like `returns a list of available algorithm names.`